Build failure on Windows due to backslashes in generated import paths

<img width="1573" height="518" alt="Image" src="https://github.com/user-attachments/assets/e396f1ab-7839-4ec6-a907-7e97ef67ce0e" />

**Problem:**
Running the build on Windows fails with `Cannot find module` errors during the compilation of the model server (e.g., `@borkdominik-biguml/uml-model-server: src/gen/langium/index.ts(11,15): error TS2307: Cannot find module './languageast.js' or its corresponding type declarations.`

**Cause:**
The error originates from the `uml-language-tooling` package. In `bin/index.ts`, the `path.relative` function is used to generate import paths for the environment-specific barrel files (`index.ts`). On Windows, this natively returns backslashes (`\`) as path separators. However, NodeJS and TypeScript require forward slashes (`/`) for module imports across all operating systems, which causes the path resolution to break. 

**Solution:**
Replace backslashes with forward slashes globally when generating the `exportPaths` array in the tooling bundle.

--- a/client/tooling/uml-language-tooling/bin/index.ts
+++ b/client/tooling/uml-language-tooling/bin/index.ts
@@ -184,7 +184,11 @@ function generateEnvIndexFiles(
         const envDir = path.join(extensionPath, env);
         const exportPaths = files
             .map(f => {
-                const rel = path.relative(envDir, f).replace(/\.tsx?$/, '.js');
+                const rel = path
+                    .relative(envDir, f)
+                    .replace(/\.tsx?$/, '.js')
+                    .split(path.sep)
+                    .join('/');
                 return `./${rel}`;
             })
             .sort();

